app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/MainNavGraph.kt

```diff
@@ -260,6 +261,7 @@ fun NavGraphBuilder.mainNavGraph(navController: NavController, activity: Compose
             snippetsComposable(this, navController)
             createSnippetComposable(this, navController)
             profileComposable(this, navController)
+            newSnippetsComposable(this)
```

Comment:
Should be able to rename this after we remove old code.  Perhaps a new PR after everything is final?